pkgs/top-level/python-packages.nix

```diff
@@ -1137,6 +1137,8 @@ in {
     enablePython = true;
   });
 
+  BoltzTraP2 = callPackage ../development/python-modules/BoltzTraP2 { };
```

Comment:
The attribute and the filename should be normalized (lowercase and with a dash instead of a underscore/point). The attribute name or `pname` should be what is needed in order to fetch the file with `fetchPypi`.

Reply:
I spent some time reading the nixpkgs manual last night. The 18.09 manuals are significantly better than 18.03 (could be that I understand more of it now). I did see that attribute names should be normalized. Thanks!